### PR TITLE
Adding some code for the website

### PR DESCRIPTION
**Pull request by cgsmith**
Fix issue #30 with this pull request.  It should work pretty well in all browsers.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<html>
+ <iframe src="http://www.easttroy.org"></iframe>
```
Comment (cgsmith) [inline index.html]: Should this be https, http, or svn+ssh?

Comment (johnjamesjacoby) [inline index.html]: <3 the simplicity

Comment (cgsmith) [inline index.html]: the web is too bloated and complicated already

**Comment by bkeating** [comment]
Looks legit. 

I'll pull it in but only because easttroy.org is possibly the only place an iframe inclusion will work :)

``` 
X-Frame-Options:SAMEORIGIN
```

#30 is on it's way. Hang tight!

**Pull request merged**